base/distributed/managers.jl

```diff
@@ -455,10 +455,10 @@ end
 const client_port = Ref{Cushort}(0)
 
 function socket_reuse_port()
-    @static if is_linux() || is_apple()
+    if Bool(ccall(:jl_has_so_reuseport, Int32, ()))
```

Comment:
`Bool` is a `Int8` (really `Int1`) type, so those wouldn't be equivalent. Although, I thought we deprecated this Bool constructor some years ago and required `!= 0` now?